hydrolib/core/basemodel.py

```diff
@@ -38,22 +75,45 @@ class FileModel(BaseModel, ABC):
 
     filepath: Optional[Path] = None
 
-    def __init__(self, filepath: Optional[Path] = None, *args, **kwargs) -> None:
+    def __init__(self, filepath: Optional[Path] = None, *args, **kwargs):
         """Initialize a model.
 
         The model is empty (with defaults) if no `filepath` is given,
         otherwise the file at `filepath` will be parsed."""
         # Parse the file if path is given
+        token = None
```

Comment:
can we rename this to `context_dir_reset_token`, or something similar, token feels quite vague to the untrained eye.